backend/migrations/2023-03-09-194135_plant_relations/up.sql

```diff
@@ -0,0 +1,70 @@
+-- Your SQL goes here
+CREATE TABLE genus (
```

Comment:
Will they also point to plant_detail so that all the information of plant_detail can be specified for a whole genus?

Reply:
exactly, check out my ER diagram in the [database_schemata.md](https://github.com/ElektraInitiative/PermaplanT/blob/d7004975419b3e62b708e99ca310ae2a6c94c870/doc/architecture/database_schemata.md#er-diagram).

I'll also update example queries with an update query for an entire genus!